docs/modules/ROOT/pages/configuration.adoc

```diff
@@ -72,6 +72,10 @@ A "rule name:term"::
 This is similar to "package name:term", but allows finer granularity by only
 applying to a particular policy rule within the package.
 
+A "collection name"::
+
+Specify a predefined collection.
```

Comment:
Should the @ prefix be mentioned here in some way?

Reply:
Yes, good catch. Added.